students/sanghyub/user/models.py

```diff
@@ -1 +1,12 @@
-from django.db import models
+from django.core.validators import validate_integer
+from django.db              import models
+
+
+class User(models.Model):
+    email        = models.EmailField(max_length= 80, null=False)
+    password     = models.CharField(max_length = 30,null=False)
+    nickname     = models.CharField(max_length = 30)
+    phone_number = models.CharField(max_length = 13) # 010 -0000-0000 try applying django.core.validators
```

Comment:
이런식으로 `=` 앞뒤 간격 전부 통일해주세요.
```suggestion
    email        = models.EmailField(max_length = 80, null = False)
    password     = models.CharField(max_length = 30, null = False)
    nickname     = models.CharField(max_length = 30)
    phone_number = models.CharField(max_length = 13) # 010 -0000-0000 try applying django.core.validators
```